Assets/Scripts/Auction/AuctionDriver.cs

```diff
@@ -78,17 +85,29 @@ private void OnDisable()
 
     private void Start()
     {
+//#if UNITY_EDITOR
+//        biddingBeginDelay = 0f;
```

Comment:
Delete or uncomment.

Reply:
Added an actual delay of 2 seconds instead to speed up dev time testing